packages/react-app/src/App.jsx

```diff
@@ -60,13 +53,14 @@ const DEBUG = true;
 const NETWORKCHECK = true;
 const USE_BURNER_WALLET = true; // toggle burner wallet feature
 const USE_NETWORK_SELECTOR = false;
+const POLL_TIME = 5000;
```

Comment:
`RPC_POLL_TIME` could be a better name.

Also would be nice to explain what this is, and that if you set it to 0 it'll make RPC requests onblock

Reply:
ok will update